stats.go

```diff
@@ -312,6 +327,20 @@ func newPoint(v *view.View, row *view.Row, start, end time.Time) *monitoringpb.P
 	}
 }
 
+func newGaugePoint(v *view.View, row *view.Row, start, end time.Time) *monitoringpb.Point {
```

Comment:
Looks like `start` is not used, does it need to be a parameter?

Reply:
Nope.. Simply make it looks symmetric.  I would delete it.